.github/renovate.json

```diff
@@ -1,11 +1,20 @@
 {
-  "enabledManagers": ["helm-requirements","gradle-wrapper", "terraform"],
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
you'll need to delete .github/dependabot.yml if you take this change as is.
previously it was selectively enabling managers and now it has them all enabled